portal/static/js/profile.js

```diff
@@ -1939,157 +1948,126 @@
             updateTreatment: function(data) {
                 var treatmentCode = this.modules.tnthAjax.hasTreatment(data);
                 if (treatmentCode) {
-                    if (String(treatmentCode) === String(this.modules.SYSTEM_IDENTIFIER_ENUM.CANCER_TREATMENT_CODE)) {
-                        $("#tx_yes").prop("checked", true);
+                    var hasCancerTreatment = String(treatmentCode) === String(this.modules.SYSTEM_IDENTIFIER_ENUM.CANCER_TREATMENT_CODE);
+                    $("#tx_yes").prop("checked", hasCancerTreatment);
+                    $("#tx_no").prop("checked", !hasCancerTreatment);
+                }
+            },
+            onBeforeInitClinicalQuestionsSection: function() {
+                if (this.mode === "profile") {
```

Comment:
If you're not going to use an `else`, it might be simpler to exit. Then you wouldn't need to indent the whole next section
```
if (this.mode !== "profile") { return; }
```